public/app/features/dashboard-scene/scene/DashboardSceneRenderer.tsx

```diff
@@ -43,13 +44,14 @@ export function DashboardSceneRenderer({ model }: SceneComponentProps<DashboardS
     <Page navModel={navModel} pageNav={pageNav} layout={PageLayoutType.Custom}>
       {editPanel && <editPanel.Component model={editPanel} />}
       {!editPanel && (
-        <CustomScrollbar autoHeightMin={'100%'}>
-          <div className={styles.canvasContent}>
-            <NavToolbarActions dashboard={model} />
-            {controls && <controls.Component model={controls} />}
-            {isEmpty ? emptyState : withPanels}
-          </div>
-        </CustomScrollbar>
+        <div className={cx(styles.pageContainer, isScopesExpanded && styles.pageContainerExpanded)}>
+          {scopes && <scopes.Component model={scopes} />}
+          <NavToolbarActions dashboard={model} />
+          {controls && <controls.Component model={controls} />}
+          <CustomScrollbar autoHeightMin={'100%'} className={styles.scrollbarContainer}>
+            <div className={styles.canvasContent}>{isEmpty ? emptyState : withPanels}</div>
+          </CustomScrollbar>
+        </div>
```

Comment:
@bfmatei any chance we rendered scopes as a common object in DashboardSceneRenderer, rather than rendering it here AND in the PanelEditor object? I assume we would have to make the `<div className={styles.pageContainer}>` a common thing for both panel edit and a view dashboard, but looking at the css for those containers they are identical, so should not cause issues.

Reply:
At first glance this was a lil tricky to do because the panel editor renderer actually contains some wrapping divs for the splitter and the controls and everything else is rendered in one of those divs 😞 . I'll try to look more into it and see if I can achieve it tho'